template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Startup.cs

```diff
@@ -48,6 +60,24 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
+#if (IndividualLocalAuth)
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+                app.UseDatabaseErrorPage();
+                app.UseDevelopmentCertificateErrorPage(Configuration);
+                //app.UseBrowserLink();
```

Comment:
It was commented out already because it was causing problems. I believe @mlorbetske is trying to get it removed completely.